docs/workers.md

```diff
@@ -210,7 +210,7 @@ expressions:
     ^/_matrix/federation/v1/get_groups_publicised$
     ^/_matrix/key/v2/query
     ^/_matrix/federation/unstable/org.matrix.msc2946/spaces/
```

Comment:
Shouldn't this one get `v1` too in that case?

Reply:
No, this endpoint was not stabilized and will be removed in the future.

Reply:
Cool, thanks!

Reply:
No problem! It was still being hit fairly actively on matrix.org last time I checked unfortunately. It was essentially replaced with the `/hierarchy` version.

#10946 tracks this work for cross-referencing.